caikit/runtime/service_factory.py

```diff
@@ -266,3 +268,43 @@ def _get_and_filter_modules(
             excluded_modules,
         )
         return clean_modules
+
+
+def get_inference_request(
+    class_name: Type[Union[ModuleBase, TaskBase]],
```

Comment:
@gabe-l-hart @joerunde does this name make sense? I have a feeling a `name` in the parameter suggests a `string` type, but I couldn't think of something that includes both `module` and a `task`

Reply:
hmmm, `honest_and_complete` naming spec might be `task_or_module_class`